docs/CHANGELOG.asciidoc

```diff
@@ -112,7 +112,7 @@ Fleet::
 Lens & Visualizations::
 Fixes normalizeTable performance bottleneck in *Lens* {kibana-pull}135792[#135792]
 
-[[release-notes-8.3.2]]
+[id="release-notes-8.3.2",canonical-url="https://www.elastic.co/guide/en/kibana/current/release-notes.html"]
```

Comment:
> [id="release-notes-8.3.2",canonical-url="https://www.elastic.co/guide/en/kibana/current/release-notes.html"]

Should we use a shared attribute for the URL in case it changes? Or are these definitely stable?

Reply:
https://www.elastic.co/guide/en/kibana/current/release-notes.html is the stable page, and all subpages change for each release. I think we're okay here.